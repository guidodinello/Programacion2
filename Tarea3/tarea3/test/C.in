# Prueba C
estaDefinidaActual
agregarAIterador 5
reiniciarIterador
estaDefinidaActual
avanzarIterador
estaDefinidaActual
insertarEnBinario (2,2.2)
insertarEnBinario (3,3.3)
nivelEnBinario 3
sumaUltimosPositivos 2
izquierdo
derecho
imprimirBinario
cantidadBinario
esAvl
mayor
removerMayor
cantidadBinario
esAvl
sumaUltimosPositivos 3
Fin