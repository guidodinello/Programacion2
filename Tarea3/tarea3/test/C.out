1>#  Prueba C.
2>Actual de it NO está definida.
3>Agregando.
4>Reiniciado.
5>Actual de it está definida.
6>Avanzando.
7>Actual de it NO está definida.
8>Insertado.
9>Insertado.
10>
11>5.50
12>izquierdo de b es vacío.
13>(3,3.30)
14>
-(3,3.30)
(2,2.20)
15>2.
16>AVL.
17>(3,3.30)
18>Removido el mayor.
19>1.
20>AVL.
21>2.20
22>Fin.
